Cargo.toml

```diff
@@ -40,18 +35,19 @@ rand = "0.8.5"
 dotenvy = "0.15.7"
 url = "2.4.0"
 mime = "0.3.17"
-regex = "1.9.0"
+regex = "1.9.1"
 serde_regex = "1.1.0"
-lazy-regex = "2.5.0"
+lazy-regex = "3.0.0"
 humantime = "2.1.0"
 humantime-serde = "1.1.1"
 glob = "0.3.1"
 ring = "0.16.20"
 hotwatch = "0.5.0"
-shuttle-actix-web = { version = "0.20.0", optional = true }
-shuttle-runtime = { version = "0.20.0", optional = true }
-shuttle-static-folder = { version = "0.20.0", optional = true }
+shuttle-actix-web = { version = "0.21.0", optional = true }
```

Comment:
I didn't intentionally mean to bump the version of these -- if they should remain the version they were on let me know and I will make sure to change them back.

Reply:
Yup, please revert these changes back 🐻

Reply:
Ok, they should be reverted to only the necessary changes now.